src/settings/ExchangesSettings.tsx

```diff
@@ -0,0 +1,358 @@
+          disabled={index === 0} // Disable if already at the top
+        >
+          <Tooltip title={t('SETTINGS.UP')}>
+            <ArrowUpwardIcon />
+          </Tooltip>
+        </IconButton>
+        <IconButton
+          sx={{ color: panelColor }}
+          onClick={() => handleMoveDown(index)}
+          disabled={index === exchangesListLength - 1} // Disable if already at the bottom
+        >
+          <Tooltip title={t('SETTINGS.DOWN')}>
+            <ArrowDownwardIcon />
+          </Tooltip>
+        </IconButton>
+      </Stack>
+      <TextField
+        value={exchangeInstructions}
+        label={t('SETTINGS.EXCHANGES.INSTRUCTIONS')}
+        multiline
+        inputProps={{ maxLength: 400 }}
+        onChange={(e) => onChange(index, 'chatbotInstructions', e.target.value)}
+      />
+      <TextField
+        value={exchangeCue}
+        label={t('SETTINGS.EXCHANGES.CUE')}
+        multiline
+        inputProps={{ maxLength: 400 }}
+        onChange={(e) => onChange(index, 'participantCue', e.target.value)}
+      />
+      <Stack direction="row" spacing={2} alignItems="center">
+        <Avatar src={exchangeAssistant.imageUrl}>
+          {exchangeAssistant.name.slice(0, 2)}
+        </Avatar>
+        <FormControl fullWidth>
+          <InputLabel>{t('SETTINGS.EXCHANGES.ASSISTANT')}</InputLabel>
+          <Select
+            value={exchangeAssistant.id}
+            renderValue={(selectedId) =>
+              assistants.assistantsList.find(
+                (assistant) => assistant.id === selectedId,
+              )?.name || selectedId
+            }
+            label={t('SETTINGS.EXCHANGES.ASSISTANT')}
+            onChange={(e) =>
+              onChange(
+                index,
+                'assistant',
+                assistants.assistantsList.find(
+                  (assistant) => assistant.id === e.target.value,
+                ) || exchangeAssistant,
+              )
+            }
+          >
+            {assistants.assistantsList.map((assistant, nb) => (
+              <MenuItem key={nb} value={assistant.id}>
+                <Avatar src={assistant.imageUrl} sx={{ mx: '1%' }}>
+                  {assistant.name.slice(0, 2)}
+                </Avatar>
+                {assistant ? (
+                  assistant.name || assistant.id
+                ) : (
+                  <Alert
+                    severity="warning"
+                    sx={{
+                      display: 'flex',
+                      justifyContent: 'center',
+                    }}
+                  >
+                    {t('SETTINGS.EXCHANGES.CREATE_ASSISTANT')}
+                  </Alert>
+                )}
+              </MenuItem>
+            ))}
+          </Select>
+        </FormControl>
+      </Stack>
+      <TextField
+        value={exchangeFollowUpQuestions}
+        type="number"
+        inputProps={{ min: 0, max: 400 }} // Min and max values for follow-up questions
```

Comment:
These numbers should be factored out to a `constants.ts` or `config.ts` file.

Reply:
added:
export const MIN_FOLLOW_UP_QUESTIONS: number = 0;
export const MAX_FOLLOW_UP_QUESTIONS: number = 400;

to new
/config/constants.ts